app/cordova.js

```diff
@@ -0,0 +1 @@
+<!-- to be generated during Cordova build -->
```

Comment:
Can you make it a regular comment? It seems that [HTML style comments are not supported by Microsoft browsers](https://kangax.github.io/compat-table/es6/#test-HTML-style_comments).